TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -250,6 +255,29 @@ public bool TryGetOpenedDocumentContext(Uri fileUri, out DocumentContext openedD
             }
         }
 
+        /// <summary>
+        /// Load Custom Analyzers
+        /// </summary>
+        /// <param name="customAnalyzerFiles"></param>
+        internal void LoadCustomAnalyzers(List<string> customAnalyzerFiles)
+        {
+            if (customAnalyzerFiles != null)
```

Comment:
You can replace this non-nullity test with an assert as we always instantiate the list in TypeCobolServerHost

Reply:
don in 5ec844a